ex1/ex1.c

```diff
@@ -8,7 +8,16 @@
 
 int main(int argc, char *argv[])
 {
-    // Your code here
+    int x = 5;
+    printf("%d: the value of x is %d\n", getpid(), x);
+
+    fork();
```

Comment:
You should assign fork to a variable to check which process you're on (child or parent). If you don't, both processes will just run the same code. Here's an example:

```
int rc = fork(); 

if (rc < 0) {
less than 0 means the fork failed. Exit here.
} else if (rc == 0) {
0 means we're in the child. Do something to the child here.
} else {
anything above 0 means we're in the parent. Do something to the parent here.
}
```